x-pack/test/api_integration/apis/transform/delete_transforms.ts

```diff
@@ -255,8 +259,47 @@ export default ({ getService }: FtrProviderContext) => {
         expect(body[transformId].destIndexDeleted.success).to.eql(false);
         expect(body[transformId].destIndexPatternDeleted.success).to.eql(true);
         await transform.api.waitForTransformJobNotToExist(transformId);
-        await transform.api.waitForIndicesToExist(destinationIndex);
```

Comment:
Why did you remove that check? I think it's still good to know that the dest index wasn't deleted.